tenseijs/tensei
# Complete rest endpoints

### How to test
- run `cd examples/blog && yarn run watch`
- make a rest API TO `api/posts?filters[av_cpc:lt]=300&perPage=4&page=1&popu[populate][0][relation]=users&popu[populate][0][limit]=5&popu[populate][0][offset]=1&popu[populate][0][populate][0][relation]=posts&popu[populate][0][populate][0][populate][0][relation]=user&popu[populate][0][populate][0][populate][0][populate][0][relation]=posts&where[$and][0][av_cpc][$lt]=30000&where[$and][1][category][$eq]=pg`
- view your console to see a sample of the info object generated
